src/common/server.ts

```diff
@@ -44,6 +44,11 @@ async function createServer(
   // Set notification type
   newEnv.LS_SHOW_NOTIFICATION = settings.showNotifications;
 
+  // Set experimental formatter capabilities
+  if (settings.enableExperimentalFormatter) {
+    newEnv.RUFF_EXPERIMENTAL_FORMATTER = "1";
+  }
```

Comment:
Does this require restarting VS Code after toggling the setting? If so, we should document this in the setting.

Reply:
No, it doesn't require restarting VS Code in my testing. (See the test plan above for a description of what it looks like in practice.)